packages/serverless/src/gcpfunction/http.ts

```diff
@@ -1,18 +1,30 @@
-import { captureException, flush, getCurrentHub, Handlers, startTransaction } from '@sentry/node';
+import {
+  addRequestDataToEvent,
+  AddRequestDataToEventOptions,
+  captureException,
+  flush,
+  getCurrentHub,
+  startTransaction,
+} from '@sentry/node';
 import { extractTraceparentData } from '@sentry/tracing';
 import { isString, logger, parseBaggageString, stripUrlQueryAndFragment } from '@sentry/utils';
 
 import { IS_DEBUG_BUILD } from '../flags';
 import { domainify, getActiveDomain, proxyFunction } from './../utils';
 import { HttpFunction, WrapperOptions } from './general';
 
-type ParseRequestOptions = Handlers.ParseRequestOptions;
-
-export interface HttpFunctionWrapperOptions extends WrapperOptions {
-  parseRequestOptions: ParseRequestOptions;
+// TODO (v8): Remove this
+interface OldHttpFunctionWrapperOptions extends WrapperOptions {
```

Comment:
This applies to the other items that are todo v8. Would you mind updating https://github.com/getsentry/sentry-javascript/issues/5194 when you merge this in that just lists what things will be removed (and how they might be replaced). If we do this now, it'll be much less work for us to figure it out for v8, we can just look at the issue and check off the items.

Reply:
Done.